website/docs/software/android/installation.md

```diff
@@ -0,0 +1,22 @@
+---
+id: android-installation
+title: Android application installation
+sidebar_label: Installation
+---
+Our Android application is available to download on Google Play.
+
+[<img alt="Download at https://play.google.com/store/apps/details?id=com.geeksville.mesh" src="https://play.google.com/intl/en_us/badges/static/images/badges/en_badge_web_generic.png" style={{zoom:'35%'}} />](https://play.google.com/store/apps/details?id=com.geeksville.mesh&referrer=utm_source%3Dgithub-homepage)
```

Comment:
Could this be centered on the page?

Reply:
You like giving me a challenge! It's easier to do in straight html than trying to work out centring something in markdown....

https://meshtastic-git-android-apt105.vercel.app/docs/software/android/android-installation